### ISSUE TITLE
create: java.nio.file.AccessDeniedException is thrown at program induced file creation

### ISSUE THREAD

**Issue by infeo** [cryptomator/fuse-nio-adapter#5]
## Basic Info
OS: Windows 10, Ver. 1709, Build 16299.64
fuse-nio-adapter: commit 2073845c37462276b335a6300d946efc706a1d27 

## Description
Creating Files not via explorer results in a I/O-Error thrown by windows and an  `java.nio.file.AccessDeniedException` thrown by the adapter (visible in errorlog)

howto:
1. start adapter (e.g. mount `X:\test` at `J:`)
1. open notepad.exe, write something into file
1. save file at `J:`

## Attachments (optional)
```
java[TID=163c]: FFFFAC019A4702A0: >>Create [UT---C] "\asd.txt", FILE_OPEN_IF, CreateOptions=60, FileAttributes=0, Security=NULL, AllocationSize=0:0, AccessToken=00000000000008C8, DesiredAccess=12019f, GrantedAccess=0, ShareAccess=3
java[TID=20f0]: FFFFAC019A61CEA0: <<QueryDirectory IoStatus=c0000034[0]
java[TID=20f0]: FFFFAC019D25F280: >>Close 0000000000000000:000001FE3B68CBE0
java[TID=20f0]: FFFFAC019D25F280: <<Close IoStatus=0[0]
java[TID=20f0]: FFFFAC019D25F280: >>Create [UT---C] "\", FILE_OPEN, CreateOptions=21, FileAttributes=10, Security=NULL, AllocationSize=0:0, AccessToken=0000000000000934, DesiredAccess=100000, GrantedAccess=0, ShareAccess=3
[Thread-164] ERROR org.cryptomator.frontend.fuse.ReadWriteFileHandler - Error opening file.
java.nio.file.AccessDeniedException: Y:\test\asd.txt
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:89)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newFileChannel(WindowsFileSystemProvider.java:116)
	at java.base/java.nio.channels.FileChannel.open(FileChannel.java:292)
	at org.cryptomator.frontend.fuse.OpenFile.<init>(OpenFile.java:27)
	at org.cryptomator.frontend.fuse.OpenFileFactory.open(OpenFileFactory.java:52)
	at org.cryptomator.frontend.fuse.ReadWriteFileHandler.createAndOpen(ReadWriteFileHandler.java:55)
	at org.cryptomator.frontend.fuse.ReadWriteAdapter.create(ReadWriteAdapter.java:67)
	at ru.serce.jnrfuse.AbstractFuseFS.lambda$init$14(AbstractFuseFS.java:194)
	at jnr.ffi.provider.jffi.NativeClosureProxy$$impl$$17.invoke(Unknown Source)
java[TID=163c]: FFFFAC019A4702A0: <<Create IoStatus=c0000185[0]
java[TID=20f0]: FFFFAC019D25F280: <<Create IoStatus=0[1] UserContext=0000000000000000:000001FE3B68D240, GrantedAccess=100000, FileInfo={FileAttributes=10, ReparseTag=0, AllocationSize=0:1000, FileSize=0:1000, CreationTime=2017-11-27T17:24:19.924Z, LastAccessTime=2017-12-13T20:45:24.231Z, LastWriteTime=2017-12-13T20:45:24.231Z, ChangeTime=1970-01-01T00:00:00.000Z, IndexNumber=0:0}
java[TID=20f0]: FFFFAC019B662EA0: >>Close 0000000000000000:000001FE3B68D240
java[TID=20f0]: FFFFAC019B662EA0: <<Close IoStatus=0[0]```

**Comment by overheadhunter** [comment]
Please retest with latest version, we fixed a bug in getattr, that lead to blocked directories, because the directory stream was not correctly closed.

**Comment by infeo** [comment]
resolved by commit 44fd7d9f89c52a836af09a87004a7771f762d684
